docs/source/building/meta-yaml.rst

```diff
@@ -597,11 +641,16 @@ need to put the same selector on multiple lines.
      url: http://path/to/unix/source        # [unix]
      md5: 88510902197cba0d1ab4791e0f41a66e  # [unix]
 
+
+NOTE: The fully rendered recipe file will be written back to the directory as ``meta.yaml.rendered``.
```

Comment:
Ok, I see. I'll update the description in the metadata section

https://github.com/conda/conda-docs/pull/297/commits/748a69d8a4f7467dd7bf68a4b3aca8364ab5bfbb